### ISSUE TITLE
Login issue for banned players

### ISSUE THREAD

**Issue by EvilOlaf** [vik1395/BungeeKick-Minecraft#1]
Hey there,

I am actually not sure if this is BungeeKick-related but for now I do not have a better guess.

tl;dr: Banned players not getting disconnected properly.

Long version:
Let me explain my issue.
For handling player punishment on our Bungee network we are using BanManager (https://github.com/BanManagement/BanManager) with local database only configuration. That means a ban issued on any server within the network will apply on all servers (after some delay of the synchronization process). 

The result is that the player got kicked from the actual server and is teleported from BungeeKick to the configured target server, e.g. lobby or hub or whatever. Then after the next sync cycle is over the player got banned from the rest of the servers including the lobby server. The player gets properly disconnected.

Now comes the fun part: When a banned player trying to login to the servers trough Bungee the client shows a "logging in...." window for a while and then got disconnected with the reason
`Internal Exception: io.netty.handler.timeout.ReadTimeoutException`

The according Bungeecoord logs showing this:
```
16:49:16 [INFORMATION] [/xxx.xxx.xxx.xxx:49945] <-> InitialHandler has connected
16:49:18 [INFORMATION] [Olaf] <-> ServerConnector [freebuild] has connected
16:49:18 [INFORMATION] [Olaf] <-> ServerConnector [lobby] has connected
16:49:18 [INFORMATION] [Olaf] <-> ServerConnector [freebuild] has disconnected
16:49:18 [INFORMATION] [Olaf] <-> ServerConnector [lobby] has disconnected
16:49:38 [INFORMATION] [Olaf] -> UpstreamBridge has disconnected
```
The default server in Bungeecords `config.yml` is `freebuild` where Bungee tries to put the player there. The player gets disconnected from `freebuild` because he is banned. BungeeKick catches the player and tries to put him to the `lobby`. He gets disconnected there as well. Then a send try seems to start and then, well, nothing happens anymore.

Thanks for reading.
